.github/workflows/main.yml

```diff
@@ -0,0 +1,164 @@
+      - name: Build
+        run: ./script/makeinstaller.sh "${{ steps.installer.outputs.filename }}"
+      - name: Upload artifact
+        if: github.repository == 'streamlink/streamlink' && (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/') || github.event_name == 'schedule')
+        uses: actions/upload-artifact@v2-preview
+        with:
+          name: ${{ steps.installer.outputs.filename }}-win32
+          path: ${{ env.STREAMLINK_DIST_DIR }}/${{ steps.installer.outputs.filename }}.exe
+
+  deploy-documentation:
+    name: Deploy docs
+    if: github.repository == 'streamlink/streamlink' && github.event_name == 'push' && (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/'))
+    needs:
+      - test
+      - documentation
+      - windows-installer
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 300
+      - name: Fetch tags
+        run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          ./script/install-dependencies.sh
+          python -m pip install -r docs-requirements.txt
+      - name: Build
+        run: make --directory=docs html
+      - name: Deploy
+        env:
+          DOCS_KEY_PASSPHRASE: ${{ secrets.DOCS_KEY_PASSPHRASE }}
+        run: ./script/deploy-docs.sh
+
+  release:
+    name: New release
+    if: github.repository == 'streamlink/streamlink' && github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')
+    needs:
+      - deploy-documentation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 300
+      - name: Fetch tags
+        run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          ./script/install-dependencies.sh
+          python -m pip install --upgrade wheel twine
+      - name: Installer file name
+        id: installer
+        run: echo ::set-output name=filename::streamlink-$(python setup.py --version | sed 's/+/_/')
+      - name: Download installer artifact
+        uses: actions/download-artifact@v2-preview
+        with:
+          name: ${{ steps.installer.outputs.filename }}-win32
+          path: ${{ env.STREAMLINK_DIST_DIR }}
+      - name: sdist and wheels
+        env:
+          RELEASE_KEY_PASSPHRASE: ${{ secrets.RELEASE_KEY_PASSPHRASE }}
+        run: ./script/build-and-sign.sh
+      - name: Github release
+        env:
+          RELEASES_API_KEY: ${{ secrets.RELEASES_API_KEY }}
+        run: ./script/github_releases.py "${STREAMLINK_DIST_DIR}/"*{.exe,.tar.gz{,.asc}}
+      - name: PyPI release
+        env:
+          PYPI_USER: ${{ secrets.PYPI_USER }}
```

Comment:
the build log is just `***` for streamlink
https://github.com/streamlink/streamlink/runs/602730028?check_suite_focus=true#step:5:12
https://github.com/streamlink/streamlink/runs/602729990?check_suite_focus=true#step:7:1

I don't think a secret is needed here

Reply:
True. I was already wondering where this was coming from. Can you send a PR with the fix, please?